analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Conversion.cs

```diff
@@ -26,7 +28,22 @@ internal sealed class Conversion : SimpleProcessor<IConversionOperationWrapper>
         operation.ToConversion();
 
     protected override ProgramState Process(SymbolicContext context, IConversionOperationWrapper conversion) =>
-        context.State[conversion.Operand] is { } value && conversion.OperatorMethod is null // Built-in conversions only
-            ? context.State.SetOperationValue(context.Operation, value)
+        context.State[conversion.Operand] is { } value
+        && PropagateSymbolValue(value, conversion) is { } propagatedValue
```

Comment:
I would swap the arguments. The `conversion` is a bigger context here, the next call have them also in that order.
```suggestion
        && PropagateSymbolValue(conversion, value) is { } propagatedValue
```